mkdocs.yml

```diff
@@ -57,6 +57,7 @@ pages:
       - 'Upgrading from previous to 0.21.0': 'admin/upgrading_crossing_0-21-0.md'
       - 'Upgrading from previous to 1.3.0': 'admin/upgrading_crossing_1-3-0.md'
       - 'Upgrading from previous to 1.5.0': 'admin/upgrading_crossing_1-5-0.md'
+      - 'Upgrading from previous to 1.14.0': 'admin/upgrading_crossing_1-14-0.md'
```

Comment:
Number (1.14.0) should be changed to the right one corresponding to the last version with metadata ID functionality, just before merging this PR.

Reply:
Fixed in 344a6bb